seccomp_linux.go

```diff
@@ -36,10 +37,9 @@ func LoadProfileFromBytes(body []byte, rs *specs.Spec) (*specs.LinuxSeccomp, err
 	return setupSeccomp(config, rs)
 }
 
-// Reload takes a Seccomp struct and a spec to reload the config
-func Reload(config *Seccomp, specgen *specs.Spec) error {
-	_, err := setupSeccomp(config, specgen)
-	return err
+// Load takes a Seccomp struct and a spec to retrieve a LinuxSeccomp
+func Load(config *Seccomp, specgen *specs.Spec) (*specs.LinuxSeccomp, error) {
```

Comment:
May `LoadProfileFromConfig()`? Would be a bit more consistent with the other `Load*` functions.

Reply:
Yes, I changed it as suggested.